addons/web/static/src/webclient/navbar/navbar.xml

```diff
@@ -75,23 +76,23 @@
           <t t-set="hotkey" t-value="undefined" />
         </t>
 
-        <t t-if="!section.childrenTree.length">
+        <t t-if="!section.childrenTree.length &amp;&amp; (!section.technical_chm || section.technical_chm == this.technical_chm.active)">
```

Comment:
!section.childrenTree.length and (!section.technical_chm || this.technical_chm.active)